build.gradle

```diff
@@ -16,24 +16,24 @@
 import org.gradle.api.internal.plugins.osgi.OsgiHelper
 
 buildscript {
-  ext.kotlinVersion = '1.2.51'
+  ext.kotlinVersion = '1.3.31'
   repositories {
 	maven { url "https://repo.spring.io/plugins-release" }
   }
   dependencies {
 	classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
 	classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
-	classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.0'
+	classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
```

Comment:
I don't think core and test builds actually use the shadow plugin, so maybe move it entirely in the `benchmarks/build.gradle` instead?

Reply:
buildscript dependencies must be top level I think or it was me just having a bad day trying to make it work.